testing/test_algorithm/test_tokenize.py

```diff
@@ -89,4 +90,12 @@ def test_prepare_dictionary():
 
 
 def test_replace_yio():
-    assert 'новье' == replace_yio('новьё')
\ No newline at end of file
+    assert 'новье' == replace_yio('новьё')
+
+
+def test_add_nltk_stopwords():
+    test_dict = {'russian': 'будто', 'english': 'yourself', 'inglesh1': 'будто'}
+    for k, v in test_dict.items():
+        stop_words = add_nltk_stopwords(set(), k)
+        if_word_in_set = v in stop_words
+        assert if_word_in_set == True
```

Comment:
In this place double check on True. `is_word_in_set` is already true or false

Reply:
It is bug, fixed

Reply:
Not shore. I mean use `assert if_word_in_set` instead `assert if_word_in_set == True`